### DISCUSSION TITLE
【求助】主题开发

### DISCUSSION THREAD

**Discussion by lkzwc** [halo-dev/halo#4692] [Q&A]
项目本地启动接口报错404，启动时没有问题的
![image](https://github.com/halo-dev/halo/assets/84896877/6e1c630a-c245-4f59-8a46-58b6bdbb8459)

**Comment by lkzwc** [comment]
![image](https://github.com/halo-dev/halo/assets/84896877/87e04122-c03e-45f9-945c-39013964cf1c)

**Comment by ruibaby** [comment]
https://docs.halo.run/developer-guide/core/run#:~:text=%E8%AF%B7%E4%B8%8D%E8%A6%81%E7%9B%B4%E6%8E%A5%E4%BD%BF%E7%94%A8%20Console%20%E7%9A%84%E8%BF%90%E8%A1%8C%E7%AB%AF%E5%8F%A3%EF%BC%883000%EF%BC%89%E8%AE%BF%E9%97%AE
